getting-started/setup-building.rst

```diff
@@ -237,6 +237,25 @@ build the :py:mod:`ctypes` module (without the flag the rest of CPython will
 still build properly).
 
 
+Optimization
+''''''''''''
+
+If you are trying to improve the performance of Python, you will probably want
+to use an optimized build of CPython. It can take a lot longer to build CPython
+with optimizations enabled, and it's usually not necessary to do so. However,
+it's essential if you want accurate benchmark results for a proposed performance
+optimization. To get an optimized build of Python,
+``configure --enable-optimizations --with-lto``.
+This sets the default make targets up to enable Profile Guided Optimization(PGO)
+and may be used to auto-enable Link Time Optimization (LTO) on some platforms.
+If you want to learn more about these options,
+please  see :ref:`Configure options <python:configure-options>`.
```

Comment:
Slightly more concise, and just as polite:

```suggestion
See :ref:`Configure options <python:configure-options>` to learn more about
these options.
```

Reply:
Untested, but you may be able to directly link to the performance options section:

```suggestion
See :ref:`Configure options <python:performance options>` to learn more about
these options.
```

Reply:
Looks like Adam's suggestion here is causing the build to fail, so I'm changing it back to

```
:ref:`Configure options <python:configure-options>`
```

Reply:
Thanks Alex!